applications/PfemFluidDynamicsApplication/custom_strategies/strategies/two_step_v_p_strategy.h

```diff
@@ -1331,14 +1333,18 @@ class TwoStepVPStrategy : public SolvingStrategy<TSparseSpace, TDenseSpace, TLin
   {
     ModelPart &rModelPart = BaseType::GetModelPart();
     ProcessInfo &rCurrentProcessInfo = rModelPart.GetProcessInfo();
+    double currentTime = rCurrentProcessInfo[TIME];
+    double timeInterval = rCurrentProcessInfo[DELTA_TIME];
     double minTolerance = 0.99999;
     bool fixedTimeStep = false;
 
     bool isItNan = false;
     isItNan = std::isnan(DvErrorNorm);
```

Comment:
@maceligueta didn't you point out that this might not work if compiled with `-ffast-math`?

Reply:
You are right, @philbucher . std::isnan does not work at all with that flag, at least with the compiler I tested. It was quite a well known issue when I googled it. Maybe they have solved this in the latest versions of the compiler.

Reply:
https://stackoverflow.com/questions/22931147/stdisinf-does-not-work-with-ffast-math-how-to-check-for-infinity

Reply:
Thank you for the interesting debate! I erased isnan and isinf in the code.